App/Containers/Contact.tsx

```diff
@@ -360,8 +357,11 @@ const mapStateToProps = (
   const adding =
     Object.keys(state.contacts.addingContacts).indexOf(address) > -1
   const directMessageThread = getDirectMessageThread(state, address)
+  const threads = getSharedThreads(state, 'name').filter(
+    thread => contact.threads.indexOf(thread.id) > -1
+  )
   return {
-    threadThumbs: getThreadThumbs(state, address, 'name'),
+    threads,
```

Comment:
no real concept of "thumbs" just the list of threads as the latest photo's thumb hash